pkg/utils/common.go

```diff
@@ -162,3 +162,20 @@ func PrintTable(writer io.Writer, header []string, rows [][]string) {
 	table.AppendBulk(rows)
 	table.Render()
 }
+
+func CleanCommitMessage(msg string) string {
```

Comment:
Do you anticipate these being used anywhere else in the code base? If not, probably best to make them local funcs in the `cmd` file. They really only apply to that command anyway.

Reply:
They are being used by the api as well